README.md

```diff
@@ -113,12 +113,12 @@ We would like you to complete the following tasks:
 ## Submission Guidance
 
 ### Shoulds
-- Only use plain Terraform in your solution
+- Use only the core functionalities provided by Terraform in your solution. This includes using modules, variables, resource blocks, etc., but excludes any external tools or libraries that extend Terraform's capabilities.
```

Comment:
I think this wording risks being too restrictive, referring to "core functionality" - possibly candidates would think that less fundamental features, such as workspaces, would be prohibited.  How about this?
```suggestion
- Use only plain Terraform in your solution, i.e. anything supported by the Terraform CLI installed by the `run.sh` script, but not external tooling or libraries that wrap or extend Terraform, such as Terragrunt or tfenv
```

Reply:
I am happy with that suggestion